### DISCUSSION TITLE
Please add Livewire support to the modules

### DISCUSSION THREAD

**Discussion by szajens** [bladewindui/ui#140] [Ideas]
Please add Livewire support to the modules:

For example, for the 'Select' component (select.blade.php):

```
@foreach ($data as $item)
    <x-bladewind::select-item label="{{ $item->{$label_key} }}"
        value="{{ $item->{$value_key} }}"
        onselect="{{ $onselect }}"
        flag="{{ ($flag_key !== '') ? $item->{$flag_key} : '' }}"
        image="{{ ($image_key !== '') ? $item->{$image_key} : '' }}"
        selected="{{ (in_array($item->{$value_key}, $selected_value)) ? 'true' : 'false' }}"
        wire:click="$set('{{ $name }}', '{{ $item->{$value_key} }}')" />
@endforeach
```
            
for select-item.blade.php:

```<div {{ $attributes->class(['py-3 pl-4 pr-3 flex items-center text-base cursor-pointer hover:bg-slate-100/90 dark:hover:bg-slate-900 dar:hover:text-slate-200 bw-select-item']) }}```

**Comment by mkocansey** [comment]
@szajens This request has come in a lot via email. 
Now that Livewire 3 is out (and is stable), I will definitely spend some time to add Livewire support. 
Apologies for the late response.

**Reply by szajens** [reply]
Thank you

**Comment by nastoychev** [comment]
@mkocansey I have seen that there is already an issue for this case. Do we have/want have to create some plan how we are going to introduce the Livewire support. I think some hours spend in management can speed up contributions and possibility to extend the library.

**Reply by mkocansey** [reply]
@nastoychev
There is currently no concrete plan honestly on how to quickly integrate Livewire support. I have actually been stuck on deciding if the Livewire version should be its own library or if I should add Livewire support to this library.

**Reply by mkocansey** [reply]
@nastoychev Theres a new [roadmap](https://bladewindui.com/roadmap) page on the docs website that gives some idea about when to expect Livewire support.

**Reply by elnasnato** [reply]
@mkocansey god bless you...I hope this can be released quickly... thank you, i love this component

**Reply by damionbroadaway** [reply]
Sorry if it's obvious. But has Livewire support been added?

The Roadmap hasn't been updated. It was slated for 3.0 and most everything else listed seems to be released.

**Reply by mkocansey** [reply]
@damionbroadaway no the Livewire support is yet to be introduced into one of the 3.x versions.

**Comment by mkocansey** [comment]
Please help me decide #185
